modules/op-disabling-the-integretion-of-tekton-hub.adoc

```diff
@@ -0,0 +1,25 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: CONCEPT
+[id="op-disabling-the-integretion-of-tekton-hub_{context}"]
+= Disabling the integration of Tekton Hub
+
+[role="_abstract"]
```

Comment:
@VeereshAradhya could you please review this part? I have no idea how it works :)

Reply:
Sure.

Reply:
This part looks good